src/components/Overview/Overview.react.js

```diff
@@ -31,31 +44,17 @@ class Overview extends Component {
   handleTitle = () => {
     this.props.history.push('/');
   };
-  // Open Video Dialog and close login and sign up dialog
+  // Open Video dialog
   handleVideo = () =>
```

Comment:
Rename it to `handleVideoModalOpen`

Reply:
Instead make a common component for handling the open/close state of Video Modal and name it `toggleVideoModal`